app/styles/main.scss

```diff
@@ -23,6 +23,17 @@ html, body {
   height: 100%;
 }
 
+.flexbox-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 100vh;
+}
+
+.flexbox-spacer {
+  flex-grow: 2;
```

Comment:
When I was testing `flex-grow`, any value greater than 1 resulted in the same output on screen, so I stuck with '2' arbitrarily because it was the nearest whole number greater than 1